smk/binning_preparation.smk

```diff
@@ -280,6 +280,16 @@ def get_assemblies_for_scaf_type(wildcards):
 
     return(asms)
 
+def calculate_batch_filesize(wildcards):
```

Comment:
Mem calculation for bwa-index uses `5GB + 22*fastasize`. We should update that to `2GB + 22*fastasize` or change the formula here to `max_job_ram_gb - 5`. Otherwise, batch size will be overestimated with 3GB more. Doesn't matter for large servers, but could matter for small servers. I am ok with fixing either one. The `5GB` baseline was arbitrary too, so `2GB` baseline could be equally good and this formula can stay as is.

Reply:
I can set it to `5GB`, but then I'm going to set a minimum of `10GB` for the config variable at parsing. (At `10GB` it would basically make single-assembly batches.)

Reply:
Done in https://github.com/arumugamlab/MIntO/pull/74/commits/5faed8b984844a782bd1600053a8c25eacb37a41